Drop-In/src/main/java/com/braintreepayments/api/DialogInteraction.java

```diff
@@ -0,0 +1,6 @@
+package com.braintreepayments.api;
+
+public enum DialogInteraction {
```

Comment:
Can this be package private and the associated callback be package private?

Reply:
Done!